Sources/Basics/FileSystem/RelativePath.swift

```diff
@@ -40,7 +40,9 @@ public struct RelativePath: Hashable, Sendable {
 
     /// Convenience initializer that verifies that the path is relative.
     public init(validating pathString: String) throws {
-        self.underlying = try .init(validating: pathString)
+        self.underlying = try .init(
```

Comment:
Same question here about being unsure we should trim whitespace for all callers of core path ops

Reply:
I reverted this change.